lib/parser.js

```diff
@@ -197,6 +200,14 @@ module.exports = function parser(identifier, string) {
     })
   }
 
+  // Consumes extend: extend google.protobuf.MessageOptions {optional string type = 50001;}
+  function consumeExtend(parent) {
```

Comment:
Sorry, I thought I remembered that I'd used `consumeX` but better to be consistent with the others.